src/main/java/objects/character/Player.java

```diff
@@ -117,19 +120,33 @@ public int getExperience() {
         return this.experience;
     }
 
-    public void addExperience(int increaseBy) {
-        // Add increaseBy experience points to the Player's experience.
-        this.experience += increaseBy;
+    public void changeExperience(int changeBy) {
```

Comment:
should this method automatically level up the player once the experience threshold has been reached?